### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by tevon456** [vayser/react-js-pagination#68]
tabindex not set on link tags by default, unable to focus when using keyboard only

**Comment by vayser** [comment]
you don't need to specify `tabindex` to access element via keyboard. It is accessible by default and with tab index you can override the order of focused elements
